spec/models/miq_server/environment_manager_spec.rb

```diff
@@ -2,14 +2,14 @@
 
 RSpec.describe "Server Environment Management" do
   let(:mac_address) { 'a:1:b:2:c:3:d:4' }
-  let(:hostname) { Socket.gethostname }
-  let(:loopback) { '127.0.0.1' }
+  let(:hostname) { Socket.gethostbyname(Socket.gethostname).first }
```

Comment:
Not related to the `local_ip_address` method but the hostname was also failing for me because this was looking for `Socket.gethostname` which returns in my case `desktop` but `Socket.gethostbyname(Socket.gethostname).first` returns the fully qualified name which in my case is something like `desktop.a.b.example.com` so even when the IP matched:

```
  1) Server Environment Management .get_network_information when in non-production mode
     Failure/Error: expect(MiqServer.get_network_information).to eq([ip_address, hostname, mac_address])
     
       expected: ["10.2.4.101", "desktop", "a:1:b:2:c:3:d:4"]
            got: ["10.2.4.101", "desktop.a.b.example", "a:1:b:2:c:3:d:4"]
```